app/lib/logic/provider_signup.rb

```diff
@@ -51,7 +51,6 @@ def signup_provider(application_plan = nil, options = {})
         #
         self.class.transaction do
           provider.save!
-          provider.update!(first_admin_id: user.id)
```

Comment:
So we are completely removing `first_admin_id ` and now only working with https://github.com/3scale/porta/tree/master/app/models/account.rb#L171?

Reply:
@guicassolato Please read this: https://github.com/3scale/porta/pull/717/files#r268259474

Reply:
I did and it's not the same. One thing is removing the attribute from the event, other is removing the field from `Account`.

Reply:
@guicassolato I know but I said that the comment applies for the entire 3rd commit, which includes removing the whole field, which was added only for that purpose as I explained there 😛 And I linked there to where was added and why 😛

Reply:
I think we should keep it for some time.

Reply:
@hallelujah Why? 😕

Reply:
Don't we need Account deleted event, for sending emails to the provider account?

Reply:
> Don't we need Account deleted event, for sending emails to the provider account

We do, but we don't need first_admin_id 😄 Later you realised yourself: 😉

> I now understand thing about first_admin_id
> I thought it was used in an email
> but in fact it was used only for Segment
> and now we remove, for real, the data in Segment we do not need it anymore

And Gui thought the same:

> I thought the same, Hery. Maybe that’s because of that recent PR to send emails to developers on plan change through API. We used first_admin there (although not first_admin_id)...
> Marta, that’s why I also bugged you on this.

So I will add a section in `Notes for reviewers(s)` explaining the whole thing in detail 😉